redisinsight/ui/src/components/base/layout/flex/flex.styles.ts

```diff
@@ -174,22 +174,24 @@ export type FlexProps = PropsWithChildren &
     full?: boolean
   }
 
-export const StyledFlex = styled.div<FlexProps>`
+export const StyledFlex = styled.div<
+  Omit<FlexProps, 'direction'> & { $direction?: (typeof dirValues)[number] }
+>`
   display: flex;
   align-items: stretch;
   flex-grow: 1;
   ${({ gap = 'none' }) => (gap ? flexGroupStyles.gapSizes[gap] : '')}
   ${({ align = 'stretch' }) => (align ? flexGroupStyles.align[align] : '')}
-  ${({ direction = 'row' }) =>
-    direction ? flexGroupStyles.direction[direction] : ''}
+  ${({ $direction = 'row' }) =>
+    $direction ? flexGroupStyles.direction[$direction] : ''}
   ${({ justify = 'start' }) =>
     justify ? flexGroupStyles.justify[justify] : ''}
   ${({ centered = false }) => (centered ? flexGroupStyles.centered : '')}
   ${({ responsive = false }) => (responsive ? flexGroupStyles.responsive : '')}
   ${({ wrap = false }) => (wrap ? flexGroupStyles.wrap : '')}
```

Comment:
Question: We prefix the props we don't want to go to the HTML with `$`, right? So why for example `wrap` or `responsive` are without it?